Project.toml

```diff
@@ -11,6 +11,7 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
 MutableArithmetics = "d8a4904e-b15c-11e9-3269-09a3773c0cb0"
 NaNMath = "77ba4419-2d1f-58cd-9bb1-8ffee604a2e3"
+PrettyTables = "08abe8d2-0d0c-5749-adfa-8a2ac140af0d"
```

Comment:
This also needs an entry in the `[compat]` section.

Reply:
Yes right!!